solution_architecture.md

```diff
@@ -269,9 +269,24 @@ If the back end calls from the mobile applications cannot be spread as evenly as
 
 ## Cross-Border Interoperability
 
-A definite prerequisite for compatibility is that the identifiers of the mobile devices can be matched, i.e. the framework by Apple and Google is being used.
+A definite prerequisite for compatibility is that the identifiers of the mobile devices can be matched, i.e. the GAEN framework by Apple and Google is being used.
 
-Further details will be added as soon as they are available.
+Most European countries are developing similar contact tracing apps, these app may use the common frameworks by Google and Apple, enabling transmission and detection of GAEN format diagnosis keys between devices running different contact tracing applications. 
```

Comment:
Suggested rewording:
Most European countries are developing similar contact tracing apps. These apps transmit and detect GAEN format diagnosis keys through the use of the common frameworks developed by Google and Apple.

Reply:
This change makes the assumption all EU countries will follow the standard GAEN framework, I presume they will but didn't want to declare it a certainty.

Reply:
The list [Mobile contact tracing apps in EU Member States](https://ec.europa.eu/info/live-work-travel-eu/health/coronavirus-response/travel-during-coronavirus-pandemic/mobile-contact-tracing-apps-eu-member-states_en) from the EU Commission shows which countries are using the GAEN framework in the column labelled "Interoperable - is this app potentially interoperable?"

Most countries list "Yes", only France and Hungary currently list "No" and there are five countries which have no app.

You could change "are developing" to "have developed" and you might like to add the link to the EU Commission's page into the document.

Reply:
I have included the link you suggested, I would prefer to keep "are developing" as there are development changes ongoing with all apps and on the iOS Android side.